### ISSUE TITLE
New idea about adding a ROI loss to improve the results

### ISSUE THREAD

**Issue by zzhan127** [junyanz/pytorch-CycleGAN-and-pix2pix#1300]
Hi everyone,

I tried a loss function to improve the results. I used masks to get the L1 loss for only the ROI (in my medial imaging area, tumor) of the image. However, the results doesn't have any improvements. Do you have any idea why this happened?

Thanks!

**Comment by junyanz** [comment]
Could you share with us more details? For example, are you trying to improve pix2pix or CycleGAN? Which aspects of the model are you trying to improve? 
Regarding the mask ideas, several papers have used the idea to improve the quality and localization of the results. You may want to check them out. See [1](https://proceedings.neurips.cc/paper/2018/file/4e87337f366f72daa424dae11df0538c-Paper.pdf) and [2](https://openaccess.thecvf.com/content_ECCV_2018/papers/Liang_Generative_Semantic_Manipulation_ECCV_2018_paper.pdf).

**Comment by improgress** [comment]
i have the same problem!  try to add a roi loss on my abjective!

**Comment by improgress** [comment]
i have the same problem! try to add a roi loss on my objective!

**Comment by junyanz** [comment]
@improgress any details?
